x/inflation/types/inflation_calculation.go

```diff
@@ -40,5 +41,9 @@ func CalculateEpochMintProvision(
 		sdk.NewDec(epochsPerPeriod).BigInt(),
 	)
 	epochProvision := sdk.NewDecFromBigInt(decEpochProvision)
+
+	// Multiply epochMintProvision with power reduction (10^18) as the calculation
+	// is based on `evmos` and the issued tokens need to be given in `aevmos`
+	epochProvision = epochProvision.Mul(sdk.NewDec(ethermint.PowerReduction.Int64()))
```

Comment:
Nit, you can use sdk.PowerReduction as it's a variable that is set at initialization. This would allow us to have modularity to support multiple power reductions

Reply:
Nice, I used  `sdk.DefaultPowerReduction` and tested it on a local node to make sure it's set to 10^18 for evmos.